### PR TITLE
doc updates

### PR DESCRIPTION
**Pull request by merlinran**
This PR:
1. mainly replace anything "broker" related. Given that we don't have an official name yet, I chose "auctioneer" because that's what bidbot interacts with, hence representing the view seen by storage providers. better names are welcome!
2. replace miners with storage providers for a few cases missed last time.
3. stop (incorrectly) mentioning IPFS nodes, URLs / IPFS multiaddrs instead.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,60 +28,60 @@ cd bidbot
...
+The `bidbot` talks to the Auctioneer through libp2p pubsub topics. This is the same technology used by many `libp2p` applications such as `lotus`. Through these topics, your `bidbot` daemon will receive new *auctions* and can send *bids* to participate.
 
-Whenever you win an auction, your `lotus-miner` will receive the deal proposal for the corresponding data so you can accept it, and a *ProposalCid* is generated. After acceptance, your `bidbot` will automatically download the *PayloadCid* data from the SB's IPFS nodes, and import it as you generally do with offline deals, simulating the following commands you might be aware of: `lotus-miner storage-deals import-data <dealCid> <carFilePath>`
+Whenever you win an auction, your `lotus-miner` will receive the deal proposal for the corresponding data so you can accept it, and a *ProposalCid* is generated. After acceptance, your `bidbot` will automatically download the *PayloadCid* data from the URL or IPFS multiaddrs given, and import it as you generally do with offline deals, simulating the following commands you might be aware of: `lotus-miner storage-deals import-data <dealCid> <carFilePath>`
```
Comment (carsonfarmer) [inline README.md:82]:
```suggestion
Whenever you win an auction, your `lotus-miner` will receive the deal proposal for the corresponding data so you can accept it, and a *ProposalCid* is generated. After acceptance, your `bidbot` will automatically download the *PayloadCid* data via the URL or IPFS multiaddrs given by the client, and import it directly from the client as you generally do with offline deals, simulating the following commands you might be aware of: `lotus-miner storage-deals import-data <dealCid> <carFilePath>`
```

**Review by carsonfarmer — APPROVED** [review]
Looks great! I made one "suggestion" to make it explicit that the data is coming directly from the client.

File: README.md (hunk 1 above)
Comment (merlinran) [reply README.md:82]: Nice! bidbot doesn't import file "directly from the client" though. I'll merge your change of the download part.

Comment (carsonfarmer) [reply README.md]: ah right 🤦 !

**Pull request merged**